### PR TITLE
feature/trigger_manager_expected_count

### PR DESCRIPTION
**Pull request by Frickerson**

### CODE REVIEW

File: Source/GameBaseFramework/Private/Gameplay/Components/GBFTriggerManagerComponent.cpp [hunk 1]
```diff
@@ -11,6 +11,11 @@ bool UGBFTriggerManagerActivationPolicy::CanActivateTrigger_Implementation( UObj
     return false;
 }
 
+int UGBFTriggerManagerActivationPolicy::GetExpectedActorsCount( const UObject * world_context, TSubclassOf< AActor > detected_actor_class ) const
```
Comment (TheEmidee) [inline Source/GameBaseFramework/Private/Gameplay/Components/GBFTriggerManagerComponent.cpp]: const TSubclassOf &

Comment (TheEmidee) [inline Source/GameBaseFramework/Private/Gameplay/Components/GBFTriggerManagerComponent.cpp]: comment the arg names if not used

**Review by TheEmidee — CHANGES_REQUESTED** [review]

File: Source/GameBaseFramework/Private/Gameplay/Components/GBFTriggerManagerComponent.cpp (hunk 1 above)
Comment (Frickerson) [reply Source/GameBaseFramework/Private/Gameplay/Components/GBFTriggerManagerComponent.cpp]: Done.

Comment (Frickerson) [reply Source/GameBaseFramework/Private/Gameplay/Components/GBFTriggerManagerComponent.cpp]: Done.

**Review by TheEmidee — APPROVED** [review]

**Pull request merged**